maestrowf/specification/yamlspecification.py

```diff
@@ -210,7 +210,7 @@ def _verify_variables(self):
                 logger.error(msg)
                 raise ValueError(msg)
 
-            if not value:
```

Comment:
I think so -- If I recall, the `PyYAML` parser (and I think YAML in general as a practice) will convert items to their types if there is no quote around them. So if someone slaps in a 0, it will come in as an `int(0)` (and by association 0.0 will come in as `float(0.0)`. I saw something similar when using `:` to separate time values and we ended up having to quote them because the YAML parser was converting them to an integer of seconds.

Reply:
I need to test and verify though.

Reply:
Ok -- this seemed to work; any other thoughts Jeremy?